character-details.mjs

```diff
@@ -1,4 +1,4 @@
-import { Component } from "//unpkg.com/can@5/ecosystem.mjs";
+import { StacheElement, type } from "//unpkg.com/can@pre/ecosystem.mjs";
```

Comment:
Can this use `core.mjs` now? I think it might have just been using `ecosystem` because it was using `can-observe`.

Reply:
we have to use ecosystem due to `enterEvent`